download/updateRestMappings.cfm

```diff
@@ -0,0 +1,25 @@
+<cfscript>
+    /***
+    * Run this template to add/update Restful services from within your web-context (e.g. http://localhost:8888/download/updateRestMappings.cfm ) 
+    * that will add/update the Rest Service. After running, confirm with http://localhost:8888/rest (to activate rest listings go
+    * to: Lucee Administrator -> Archives & Resources -> Rest, and activate "List services")
+    ***/
+
+    if( cgi.SERVER_NAME=="localhost") {
+        basePath=getDirectoryFromPath( getTemplatePath() );
```

Comment:
getTemplatePath() is deprecated. https://docs.lucee.org/reference/functions/gettemplatepath.html

Reply:
@cfmitrah Thanks for the help!!! I've changed the variable assignment to not get the variable from the request scope and also, got rid of the getTemplatePath().

Could you do another retry?

Reply:
@andreasRu I've made some tests on this PR. It works fine